boards/arm/lpcxpresso54114_m0/doc/lpcxpresso54114_m0.rst

```diff
@@ -36,7 +36,7 @@ You can debug an application in the usual way. Here is an example for the
 
 .. zephyr-app-commands::
    :zephyr-app: samples/subsys/ipc/ipm_mcux
-   :board: lpcxpresso54114
+   :board: lpcxpresso54114_m4
```

Comment:
> The suffix _m4 is there because the ipm_mcux sample builds both binaries for the m0 core and for the m4 core. M0 binary is embedded within m4 binary during the build process and is loaded into the m0 core at runtime from the m4 program.

Right, I forgot about that.

> Now the question is if we rename the link above to _m4.html and add "(M4 Core)" to the page title, just like "(M0 Core)" here:

I'd say yes, but let's see what @dbkinder thinks.

Reply:
Well, that's interesting, (and that's what lines 27-29 are saying too.)

How about you change the title in ``boards/arm/lpxcpresso54114_m4/doc/lpcxpresso54114.rst``  to add the ``(M4 Core)`` and we'll call it good.

    .. _lpcxpresso54114:
    
    NXP LPCXPRESSO54114 (M4 Core)
    #############################

Reply:
Changed.